core/patina_internal_cpu/src/paging/aarch64.rs

```diff
@@ -117,11 +119,11 @@ mod tests {
     fn test_remap_memory_region() {
         let mut mock_page_table = MockPageTable::new();
 
-        mock_page_table.expect_remap_memory_region().returning(|_, _, _| Ok(()));
+        mock_page_table.expect_map_memory_region().returning(|_, _, _| Ok(()));
 
         let mut paging = EfiCpuPagingAArch64 { paging: mock_page_table };
 
-        let result = paging.remap_memory_region(0x1000, 0x1000, MemoryAttributes::Uncacheable);
+        let result = paging.rap_memory_region(0x1000, 0x1000, MemoryAttributes::Uncacheable);
```

Comment:
This looks like a typo.

```suggestion
        let result = paging.remap_memory_region(0x1000, 0x1000, MemoryAttributes::Uncacheable);
```